File: PYTHON/ArrayFrequency.py
from collections import Counter

def findFrequency(inputArr):
    countObject  = Counter(inputArr)
    return dict(countObject)


inputArr = ["a", "b", "c", "b", "b", "c", "c", "c", "a", "c"]
inputArr2 = [1, 2, 3, 4, 1, 2, 3, 5, 6, 8, 9, 6 , 6]

print(findFrequency(inputArr))
print(findFrequency(inputArr2))

File: JS/ArrayFrequency.js
const inputArr = ["a", "b", "c", "b", "b", "c", "c", "c", "a", "c"];
const inputArr2 = [1, 2, 3, 4, 1, 2, 3, 5, 6, 8, 9, 6 , 6];


function findCharFrequency(inputArr) {
    const counter = {};

    inputArr.forEach(e => {
        counter[e] ? counter[e]++ : counter[e] = 1;
    })

    return counter;
}

console.log(findCharFrequency(inputArr));
console.log(findCharFrequency(inputArr2));